spec/unit/outputs/elasticsearch/http_client_spec.rb

```diff
@@ -180,6 +180,53 @@
     end
   end
 
+  describe "#bulk" do
+    subject { described_class.new(base_options) }
+
+    require "json"
```

Comment:
🤔 I don't see any of the lines in the new spec using the top-level `::JSON` constant, so either we don't need this `require "json"` statement or something is broken further down.

As a side note, because of how rspec loads things, when we _do_ have specs that require external additional dependencies, it's better to load them in a `before` block so that they're only loaded if we're _executing_ the example group:

~~~ ruby
    before do
      require "json"
    end
~~~